### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by mferno**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "springworks/babel",
```
Comment (dbrockman) [inline .eslintrc]: - [ ] Rename `.eslintrc` files to `.eslintrc.json`

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+.idea
+*.iml
+
+coverage
+
+/config/runtime.json
+
+# Ignore build output
+lib
+.eslintcache
```
Comment (dbrockman) [inline .gitignore]: - [ ] You need a `.npmignore` file with the same content except that it ignores src and test

Comment (mferno) [inline .gitignore]: Yes!

File: test/.eslintrc [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "springworks/mocha",
```
Comment (dbrockman) [inline test/.eslintrc]: - [ ] rename this one as well

**Pull request merged**